pySDC/projects/DAE/problems/transistor_amplifier.py

```diff
@@ -61,8 +61,8 @@ class one_transistor_amplifier(ptype_dae):
         Springer (2009).
     """
 
-    def __init__(self, nvars, newton_tol):
-        super().__init__(nvars, newton_tol)
+    def __init__(self, newton_tol):
+        super().__init__(nvars=(5, 0), newton_tol=newton_tol)
```

Comment:
Why is this class a DAE if there is no algebraic part? Does it make sense to switch to `mesh` for this class?

Reply:
There are also fully-implicit DAEs, i.e., you don't have any algebraic equations here, also no algebraic variables. The thing what it makes the equation to a DAE is its singular Jacobian matrix. So, a DAE mesh does not make any sense here.